### ISSUE TITLE
🐛 Bug Report: Unavailable version of @backstage/plugin-techdocs-react in dependencies of techdocs plugin

### ISSUE THREAD

**Issue by mountainaireman** [backstage/backstage#20027]
### 📜 Description

Upgrading to 1.17.5 throwing error for unavailable version of techdocs-react

`error Couldn't find any versions for "@backstage/plugin-techdocs-react" that matches "^1.1.10"`

Looks like techdocs plugin 1.7.0 on npm is showing "@backstage/plugin-techdocs-react": "^1.1.10",

### 👍 Expected behavior

Able to yarn install with an existing version of "@backstage/plugin-techdocs-react"

### 👎 Actual Behavior with Screenshots

```Some packages are outdated, updating

bumping @backstage/app-defaults in app to ^1.4.2
bumping @backstage/catalog-model in app to ^1.4.1
bumping @backstage/cli in app to ^0.22.12
bumping @backstage/core-app-api in app to ^1.9.1
bumping @backstage/core-components in app to ^0.13.4
bumping @backstage/core-plugin-api in app to ^1.5.3
bumping @backstage/plugin-analytics-module-ga in app to ^0.1.32
bumping @backstage/integration-react in app to ^1.1.18
bumping @backstage/plugin-api-docs in app to ^0.9.10
bumping @backstage/plugin-catalog in app to ^1.12.4
bumping @backstage/plugin-catalog-common in app to ^1.0.15
bumping @backstage/plugin-catalog-graph in app to ^0.2.35
bumping @backstage/plugin-catalog-import in app to ^0.9.13
bumping @backstage/plugin-catalog-react in app to ^1.8.3
bumping @backstage/plugin-github-actions in app to ^0.6.4
bumping @backstage/plugin-org in app to ^0.6.13
bumping @backstage/plugin-permission-react in app to ^0.4.14
bumping @backstage/plugin-scaffolder in app to ^1.14.4
bumping @backstage/plugin-search in app to ^1.3.6
bumping @backstage/plugin-search-react in app to ^1.6.4
bumping @backstage/plugin-tech-radar in app to ^0.6.7
bumping @backstage/plugin-techdocs in app to ^1.6.8
bumping @backstage/plugin-techdocs-module-addons-contrib in app to ^1.0.18
bumping @backstage/plugin-techdocs-react in app to ^1.1.9
bumping @backstage/plugin-user-settings in app to ^0.7.8
bumping @backstage/theme in app to ^0.4.1
bumping @backstage/test-utils in app to ^1.4.2
bumping @backstage/catalog-model in backend to ^1.4.1
bumping @backstage/cli in backend to ^0.22.12
bumping @backstage/backend-common in backend to ^0.19.4
bumping @backstage/backend-tasks in backend to ^0.5.7
bumping @backstage/catalog-client in backend to ^1.4.3
bumping @backstage/config in backend to ^1.0.8
bumping @backstage/integration in backend to ^1.6.2
bumping @backstage/plugin-app-backend in backend to ^0.3.50
bumping @backstage/plugin-auth-backend in backend to ^0.18.8
bumping @backstage/plugin-auth-node in backend to ^0.2.19
bumping @backstage/plugin-catalog-backend in backend to ^1.12.4
bumping @backstage/plugin-catalog-backend-module-gitlab in backend to ^0.2.6
bumping @backstage/plugin-catalog-backend-module-msgraph in backend to ^0.5.9
bumping @backstage/plugin-catalog-node in backend to ^1.4.3
bumping @backstage/plugin-permission-common in backend to ^0.7.7
bumping @backstage/plugin-permission-node in backend to ^0.7.13
bumping @backstage/plugin-proxy-backend in backend to ^0.3.2
bumping @backstage/plugin-scaffolder-backend in backend to ^1.16.5
bumping @backstage/plugin-search-backend in backend to ^1.4.2
bumping @backstage/plugin-search-backend-module-pg in backend to ^0.5.11
bumping @backstage/plugin-search-backend-node in backend to ^1.2.6
bumping @backstage/plugin-techdocs-backend in backend to ^1.6.7
bumping @backstage/types in backend to ^1.1.0
bumping @backstage/cli in @internal/plugin-winsql-licenses to ^0.22.12
bumping @backstage/core-app-api in @internal/plugin-winsql-licenses to ^1.9.1
bumping @backstage/core-components in @internal/plugin-winsql-licenses to ^0.13.4
bumping @backstage/core-plugin-api in @internal/plugin-winsql-licenses to ^1.5.3
bumping @backstage/theme in @internal/plugin-winsql-licenses to ^0.4.1
bumping @backstage/test-utils in @internal/plugin-winsql-licenses to ^1.4.2
bumping @backstage/config in @internal/plugin-winsql-licenses to ^1.0.8
bumping @backstage/dev-utils in @internal/plugin-winsql-licenses to ^1.0.20
bumping @backstage/cli in root to ^0.22.12

Running yarn install to install new versions ✖ 
[1/5] Validating package.json...
[2/5] Resolving packages...
error Couldn't find any versions for "@backstage/plugin-techdocs-react" that matches "^1.1.10"
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Error: Couldn't find any versions for "@backstage/plugin-techdocs-react" that matches "^1.1.10"
    at MessageError.ExtendableBuiltin (/usr/share/yarn/lib/cli.js:721:66)
    at new MessageError (/usr/share/yarn/lib/cli.js:750:123)
    at Function.<anonymous> (/usr/share/yarn/lib/cli.js:50351:13)
    at Generator.next (<anonymous>)
    at step (/usr/share/yarn/lib/cli.js:310:30)
    at /usr/share/yarn/lib/cli.js:321:13
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

Command 'yarn' exited with code 1

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### 👟 Reproduction steps

`yarn backstage-cli versions:bump`

### 🖥️ Your Environment

```yarn backstage-cli info
yarn run v1.22.19
OS:   Linux 5.15.90.1-microsoft-standard-WSL2 - linux/x64
node: v16.20.1
yarn: 1.22.19
cli:  0.22.9 (installed)
backstage:  1.17.5

Dependencies:
  @backstage/backend-app-api          0.4.5
  @backstage/backend-common           0.18.5
  @backstage/backend-dev-utils        0.1.1
  @backstage/backend-plugin-api       0.5.4
  @backstage/catalog-client           1.4.3
  @backstage/catalog-model            1.4.1
  @backstage/cli-common               0.1.12
  @backstage/config                   1.0.8
  @backstage/core-components          0.12.5
  @backstage/core-plugin-api          1.5.3
  @backstage/errors                   1.2.1
  @backstage/eslint-plugin            0.1.3
  @backstage/integration-aws-node     0.1.5
  @backstage/plugin-catalog-common    1.0.15
  @backstage/plugin-permission-common 0.7.7
  @backstage/plugin-permission-react  0.4.14
  @backstage/plugin-proxy-backend     0.2.41
  @backstage/plugin-scaffolder-node   0.1.5
  @backstage/plugin-search-common     1.2.5
  @backstage/release-manifests        0.0.9
  @backstage/theme                    0.2.19, 0.4.1
  @backstage/types                    1.1.0
  @backstage/version-bridge           1.0.4
Done in 0.45s.
```

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

No, I don't have time to work on this right now

**Comment by CiscoRob** [comment]
Same issue.

Seeing the list of packages show up with `-next.0` and no `1.1.10`.

Reset the yarn.lock to use the 1.1.9 version and try to watch the `^1.1.9` defaulted for packages in the `packagse/app` references. Also watch the `yarn.lock` to see if there is any references. You can install the explicit `yarn --cwd packages/app add @backstage/plugin-techdocs-react@1.1.9`. After skim the yarn.lock and ensure that the resolved packages use the 1.1.9.

Ultimately every minor revision release Backstage tends to break and don't trust backwards compatibility with their library. :(

**Comment by CiscoRob** [comment]
![image](https://github.com/backstage/backstage/assets/133238823/2b4dcbaa-4525-4130-88ed-097175de6116)

**Comment by ronballesteros** [comment]
Experienced this today when trying to update backstage to 1.17.

**Comment by unixunion** [comment]
Is there a workaround?

**Comment by SimonRousingBerg** [comment]
I belive it will continue to fail, because no version 1.1.10 is not available...

https://www.npmjs.com/package/@backstage/plugin-techdocs-react?activeTab=versions

**Comment by awanlin** [comment]
Hi everyone, there was an issue pushing out the `1.18.0` release. The packages got to NPM but for some reason they weren't visible or were partly visible. Patch versions of the packages have been pushed and things should be working again. Can you please try again? Follow up here if you continue to have issues

**Comment by SimonRousingBerg** [comment]
A clean new app creation works for me now.

**Comment by vinzscam** [comment]
Closing this as it was fixed with the last 1.18 release
